File: Informatyka-master/Klasa 1/Cpp/szyfr_cezara.cpp
/*
 * szyfr_cezara.cpp
 *
 */

#include <iostream>

void szyfruj(char tekst[], int klucz){
    klucz = klucz % 26;
    while(tekst[i] != '\0)'{
        if ((int)tekst[i] + klucz > 122)
            tekst[i] = (char)((int)tekst[i] + klucz - 26);
        else
            ;
        i++;
    }
}

int main(int argc, char **argv)
{
    char tekst[100];
    int klucz = 3;
    cout << "Podaj tekst: ";
    cin >> tekst;
    cout << "Podaj klucz: ";
    cin >> klucz;
    szyfruj(tekst, klucz)
    cout << tekst << endl;


	return 0;
}

File: Informatyka-master/Klasa 1/Python/szyfr_cezara.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


def szyfruj_cezar(tekst, klucz):
    klucz = klucz % 26
    szyfrogram = ''
    for znak in tekst:
        znak = znak.lower()
        if ord(znak) + klucz > 122:
            szyfrogram += chr(ord(znak) + klucz - 26)
        else:
            szyfrogram += chr(ord(znak) + klucz)
    return szyfrogram


def deszyfruj_cezar(tekst, klucz):

    return tekst


def main(args):
    tekst = raw_input("Podaj tekst: ")
    klucz = int(raw_input("Podaj klucz: "))
    szyfrogram = szyfruj_cezar(tekst, klucz)
    print szyfrogram
    print deszyfruj_cezar(szyfrogram, klucz)
    return 0


if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
